src/UI/Article.jsx

```diff
@@ -75,7 +84,21 @@ function Article(props) {
         backgroundPosition: 'center',
       }}
     >
-      <div className="absolute bottom-0 left-0 right-0 h-1/2 bg-black/30 p-4 backdrop-blur-lg">
+      <a
+        href={getLink(props)}
+        target="_blank"
+        rel="noopener noreferrer"
+        className={
+          'absolute right-2 top-2 rounded-lg bg-white/[.50] object-right opacity-100'
+        }
+      >
+        <OpenInNewIcon
+          sx={{
+            color: orange[600],
```

Comment:
Let's use the primary (yellow) color, the same as in news titles

Hint: 

If you add `text-yellow-500` to the parent (in this case `<a>`) the icon will inherit the color, and you no longer need this `sx` prop